en/micro-integrator/docs/use-cases/examples/sequence_examples/using-fault-sequences.md

```diff
@@ -78,6 +78,36 @@ the `get-property` XPath function. The following log mediator logs the actual er
 </log>
 ``` 
 
+The following is a sample of the configurations to use the Fault sequence in an API. Make note of the "faultSequence" attribute in the "resource" element.
+
+```xml
+<api context="/testFault" name="FaultTestAPI">
+    <resource faultSequence="testFault" methods="POST" uri-template="/v1">
```

Comment:
can we use default fault seq here since testFault doesn't seem to be defined

Reply:
Changed this to "fault". Please check this and see now.